regexes.go

```diff
@@ -7,6 +7,7 @@ import (
 
 const (
 	alphaRegexString                 = "^[a-zA-Z]+$"
+	alphaSpaceRegexString            = "^[a-zA-Z ]+$"
```

Comment:
Would it be better for the regex to not match if the input contains only spaces?

Reply:
I would leave it like that

But people already asked for what you suggest, so I don't know

https://github.com/go-playground/validator/issues/1191#issuecomment-2072078625

But then, should leading and trailing space are valid?

- ` foo`, trailing space, valid or invalid?
- `foo `, leading space, valid or invalid
- `` empty, invalid?
- ` ` only space, invalid?
- `foo bar`

I'm asking because the regexp will need to be adapted

Reply:
Actually, I reviewed https://github.com/go-playground/validator/issues/1191#issuecomment-2072078625 before asking the question. My question was more for clarification to ensure we’re aligned on the intended behavior. 
I think that making the regex more complex to handle edge cases like these is unnecessary. It avoids unnecessary complexity while meeting the expected use cases.